### DISCUSSION TITLE
Chia founders messed up

### DISCUSSION THREAD

**Discussion by Nichabell** [Chia-Network/chia-blockchain#6359] [General]
It is my belief that the CHIA founders messed up.  When chia went public they should have sold a ton of the pre-mine to drive the price down to what it actually costs to mine chia with 120 PiB of net space.  I believe I heard values as low as $20/XCH.  The whole point of the pre-mine was to give them the power to stabilize the market.  The fact that chia peaks around $1800 and bottoms out around $600 is apparently what will drive the net space.

At the time I wrote this netspace is around 16.134 EIB
If we assume price stabilizes around $1200/XCH
If we assume 5 year ROI is the minimum investment point
1 Plot costs me about $.21 in electricity or $2/TB
If you got storage space early you paid $15/TB (retail) no idea how cheap it is for business
I am not even sure how to estimate the cost of running a farm but lets assume is it about $1/TB/Month
That puts your 5 year all in cost at $77/(TB*5 year)
Current XCH/(TB*DAY) is .0005578 converting to XCH/(TB*5 year) is 1.017985

as you can see for $77 you can make $1221.58
I would argue this should drive the price of CHIA down to 80-85/XCH however this world is apparently not sane.

I fully expect the net space to continue to grow to about 255.96 EIB
From here CHIA is no longer a good investment for miners however that is where CHIA gets you.  Because you put in all your money up front most people will suffer from sunk cost fallacy and the need to win 2 CHIA to have a chance of making any money back  Thus Miners will lose money as the netspace continues to grow to 2,559.6 EIB

And just as a point of reference you will need about 410TB to have a decent chance of winning 2 XCH within 5 years at this level.

So you will be spending $31,570 for a high probability of winning $2,400

-edit updated for in inverted calc.

**Comment by XIII39** [comment]
> 1 Plot costs me about $4 in electricity or $40/TB

How much do you pay per kWh for electricity? I think you should think about moving!

**Reply by Nichabell** [reply]
Ops inverted
16 plots a day -> 480 plots every 30 days
$100 dollars a month to power my pc
$.21 a plot

My electricity is pretty average for US

**Comment by FSys77** [comment]
Same has been said over and over again in the early days of Bitcoin.

But: Price drives difficulty. Difficulty drives price.

Unless Chia is totally flawed, it will continously go up in price.

**Reply by caldwelljt2** [reply]
> Same has been said over and over again in the early days of Bitcoin.
> 
> But: Price drives difficulty. Difficulty drives price.
> 
> Unless Chia is totally flawed, it will continously go up in price.

imho, It's flawed. I can't articulate exactly why, beyond the price slide. If I had the time I might look closer at the last few price bumps, particularly those that don't correlate with BTC. I'll hazard a guess the trading patterns right before and during them are suspect.

**Comment by BrandtH22** [comment]
Closing as no recent responses
